about.html

```diff
@@ -0,0 +1,108 @@
+                    <li class="menu"><a href="./index.html#program">Program</a></li>
+                    <li class="menu"><a href="./index.html#speakers">Speakers</a></li>
+                    <li class="menu"><a href="./index.html#sponsor">Sponsors</a></li>
+                    <li class="about"><a href="#">About</a></li>
+                </ul>
+            </div>
+        </header>
+        <div class="intro">
+            <h2 class="hello">"The book fair comeback"</h2>
+            <h1 id="title" class="title">The book fair returns to lima and in person for the entire peruvian public.</h1>
+            <p class="description">After a long time of being locked up due to the pandemic, the book fair returns to "el Parque de los Proceres" in the district of Jesus María where you will meet your 
+                favorite authors and writers in this event that takes place every year in the July</p>
+            <h3 class="date">22 July, 2022</h3>
+            <p class="location">Lima, Perú</p>
+        </div>
+    </div>
+    <div class="main-program">
+        <div class="div-list">
+            <div class="item-list">
+                <img src="./images/list2.png" alt="conversation" class="images">
+                <h3 class="img-title">Literary Talent</h3>
+                <p class="img-des">Among the representatives of Spain, David Roas, a Spanish writer and literary critic, specializing in fantastic literature, stands out, who will share dialogue tables 
+                    on literary criticism and crime novels. Once again, we are visited by Juan Casamayor, founder of Páginas de Espuma, an independent label that is a reference in the genre of short stories in Spanish.</p>
+            </div>
+        </div>
+        <div class="space"></div>
+    </div>
+    <div  class="featured-speakers">
+        <h2 class="speakers-title">More about the event</h2>
+        <div class="linea-2"></div>
+        <h3 class="guion">if you want more information about the event go here:</h3>
+        <div class="other-programs">
+            <a href="https://elperuano.pe/noticia/163398-atencion-lectores-en-julio-vuelve-la-feria-internacional-del-libro-de-lima#:~:text=La%20espera%20termin%C3%B3%3A%20la%2026%C2%AA,avenida%20Salaverry%2C%20en%20Jes%C3%BAs%20Mar%C3%ADa." class="program">
+                <img src="./images/feria.jpg" alt="feria" class="feria-img">
+                <span class="cover"></span>
+                <h3 class="year">2022</h3>
+                <p class="sentence">26° Book fair</p>
+            </a>
+            <a href="https://librosami.pe/2019/06/feria-internacional-del-libro-de-lima-2019-autores-invitados-y-todo-lo-que-debes-saber-del-evento/" class="program">
+                <img src="./images/feria2.jpg" alt="feria" class="feria-img">
+                <span class="cover"></span>
+                <h3 class="year">2019</h3>
+                <p class="sentence">24° Book fair</p>
+            </a>
+        </div>
+    </div>
```

Comment:
- [ ] Kindly give more height to pictures to follow the design.


![image](https://user-images.githubusercontent.com/38635158/176970674-6368b5c1-7c69-4492-94e0-341c458188da.png)
___
### ORIGINAL DESIGN
![image](https://user-images.githubusercontent.com/38635158/176970693-4a3fcb4a-129c-4627-af6f-a61779bf139c.png)